geo.py

```diff
@@ -414,15 +416,15 @@ def _load_us_state_coords():
     return US_STATE_COORDS
 
 
-def coords_for_us_state_code(code):
+def coords_for_us_state_code(code: str) -> Optional[Tuple]:
```

Comment:
Probably should be `-> Optional[Tuple[float, float]]:`

Reply:
...or, even better, define `LatLonTuple = Tuple[float, float]` somewhere centrally and use it consistently for (lat, lon) tuples